README.md

```diff
@@ -43,44 +43,42 @@ This project contains 2 packages:
 [npm-java-parser-image]: https://img.shields.io/npm/v/java-parser.svg?color=blue&label=java-parser&logo=java-parser
 [npm-java-parser-url]: https://www.npmjs.com/package/java-parser
 
-## Status
-
-- Parser package alpha version done, it can parse most of Java code. However, we still need to make some tweaks and improvements.
-- The printer (actually `prettier-java` package) is mostly done, it can output formatted code but needs to be improved on some cases.
-
 ## Install
 
 ### Pre-requirements
 
-Since the plugin is meant to be used with Prettier, you need to install it:
-
-`npm install -g prettier`
-
-or
+- Node version 10+
 
-`yarn global add prettier`
+### Install Prettier and Prettier-Java plugin
 
-### Install plugin
+Since the plugin is meant to be used with Prettier, you need to install it, as well as the prettier-plugin-java package.
```

Comment:
Are you sure? As I understand, `prettier` is a transitive dependency of `prettier-plugin-java` so no need to install both.

See [here](https://github.com/hdurix/nba-stats/blob/master/package.json), I only defined `prettier-plugin-java`.

Reply:
You're right if it is installed locally. When installed globally, the prettier command is not added to the PATH if we only install prettier-plugin-java.